pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateData.java

```diff
@@ -36,16 +39,22 @@ public record ServiceUnitStateData(
         }
     }
 
+    public ServiceUnitStateData(ServiceUnitState state, String dstBroker, String sourceBroker,
+                                Map<String, Optional<String>> splitServiceUnitToDestBroker, long versionId) {
+        this(state, dstBroker, sourceBroker, splitServiceUnitToDestBroker, false,
+                System.currentTimeMillis(), versionId);
+    }
+
     public ServiceUnitStateData(ServiceUnitState state, String dstBroker, String sourceBroker, long versionId) {
-        this(state, dstBroker, sourceBroker, false, System.currentTimeMillis(), versionId);
+        this(state, dstBroker, sourceBroker, Map.of(), false, System.currentTimeMillis(), versionId);
```

Comment:
`Map.of` seems redundant. Can it be null? (applies the same for the below constructors)

Reply:
Updated.

Reply:
Please change Map.of() to null.